### PR TITLE
Facemesh: Add bounding boxes to each named keypoint in result

### PR DESCRIPTION
**Pull request by gohai**
This adds a bounding box to each named part of the face - as discussed [here](https://github.com/ml5js/ml5-next-gen/pull/42#discussion_r1317387553).

### CODE REVIEW

**Review by B2xx — APPROVED** [review]

File: src/Facemesh/index.js [hunk 1]
```diff
@@ -185,12 +185,19 @@ class Facemesh {
...
           x: keypoint.x,
           y: keypoint.y,
           z: keypoint.z,
         });
+        namedKeypoint.x = (namedKeypoint.x === undefined || keypoint.x < namedKeypoint.x) ? keypoint.x : namedKeypoint.x;
+        namedKeypoint.y = (namedKeypoint.y === undefined || keypoint.y < namedKeypoint.y) ? keypoint.y : namedKeypoint.y;
+        namedKeypoint.width = (namedKeypoint.width === undefined || keypoint.x - namedKeypoint.x > namedKeypoint.width) ? (keypoint.x-namedKeypoint.x) : namedKeypoint.width;
+        namedKeypoint.height = (namedKeypoint.height === undefined || keypoint.y - namedKeypoint.y > namedKeypoint.height) ? (keypoint.y-namedKeypoint.y) : namedKeypoint.height;
+        namedKeypoint.centerX = namedKeypoint.x + namedKeypoint.width / 2;
+        namedKeypoint.centerX = namedKeypoint.y + namedKeypoint.height / 2;
```
Comment (ziyuan-linn) [inline src/Facemesh/index.js:200]:
```suggestion
        namedKeypoint.centerY = namedKeypoint.y + namedKeypoint.height / 2;
```

**Review by ziyuan-linn — COMMENTED** [review]
Thank you @gohai! I tested this and it works great. It is really easy to get a bounding box of a face part with this interface. I just have a quick inline comment about a typo, but everything else looks good!

@shiffman I think this can be merged after the typo fix :)

File: src/Facemesh/index.js (hunk 1 above)
Comment (ziyuan-linn) [reply src/Facemesh/index.js:200]: `namedKeypoints.centerX` is set twice, changing one of them to `namedKeypoints.centerY`

Comment (gohai) [reply src/Facemesh/index.js:200]: Good catch - and thank you for reviewing this @ziyuan-linn

**Review by shiffman — APPROVED** [review]
Apologies for the long delay in reviewing this pull request, excited to merge this!

**Pull request merged**